src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/JakartaCodeActionHandler.java

```diff
@@ -178,15 +183,15 @@ public List<CodeAction> codeAction(JakartaJavaCodeActionParams params, IPsiUtils
                     if (diagnostic.getCode().getLeft().equals(ManagedBeanConstants.DIAGNOSTIC_CODE)) {
                         codeActions.addAll(ManagedBeanQuickFix.getCodeActions(context, diagnostic));
                     }
-//                    if (diagnostic.getCode().getLeft().equals(ManagedBeanConstants.DIAGNOSTIC_CODE_PRODUCES_INJECT)) {
-//                        codeActions.addAll(ConflictProducesInjectQuickFix.getCodeActions(context, diagnostic, monitor));
-//                    }
+                    if (diagnostic.getCode().getLeft().equals(ManagedBeanConstants.DIAGNOSTIC_CODE_PRODUCES_INJECT)) {
+                        codeActions.addAll(ConflictProducesInjectQuickFix.getCodeActions(context, diagnostic));
+                    }
                     if (diagnostic.getCode().getLeft().equals(ManagedBeanConstants.DIAGNOSTIC_CODE_INVALID_INJECT_PARAM)) {
-//                        codeActions.addAll(RemoveInjectAnnotationQuickFix.getCodeActions(context, diagnostic, monitor));
+                        codeActions.addAll(RemoveInjectAnnotationQuickFix.getCodeActions(context, diagnostic));
```

Comment:
For each case where more than one quick fix is being used to produce code actions for a specific diagnostic, we're going to need to make sure that each quick fix is working off it's own cloned copy of the context. Maybe not in this PR, but I'd suggest doing a pass over JakartaCodeActionHandler and making an additional copy (or copies) for each diagnostic that has more than one quick fix.

Reply:
Good point. The cases up to now have been where one of the pair of quick fixes was making its own copies but I think we should do cloning in this file (even if redundant in some cases) to ensure correctness.